02 - Community Expansions/02.01 Plugins by Category/🗂️ 02.01 Plugins by Category.md

```diff
@@ -9,50 +9,137 @@ tags:
 
 #placeholder/description 
 
+## Categories
```

Comment:
Wow - This page is fantastically useful now - thank you for categorising the categories, they are now much easier to understand!!!!

Reply:
agree, when I finally have some free time, I'd really wanna spend some time browsing this...